group_vars/hosts/telegraf.yml

```diff
@@ -48,3 +48,4 @@ telegraf_plugins:
       interfaces:
         - "bond0"
         - "bond1"
+  - name: exec
```

Comment:
Thanks for catching this and clearing up my misunderstanding.  I have added the options and have tested them.